cfgov/cfgov/urls.py

```diff
@@ -385,11 +385,11 @@ def empty_200_response(request, *args, **kwargs):
     re_path(r'^_status/', include_if_app_enabled('watchman', 'watchman.urls')),
 
     re_path(
-        r'^consumer-tools/financial-well-being/',
+        r'^(?i)consumer-tools/financial-well-being/',
         include('wellbeing.urls')
     ),
     re_path(
-        r'^about-us/diversity-and-inclusion/',
+        r'^(?i)about-us/diversity-and-inclusion/',
```

Comment:
Same as above, doesn't this need to be added to the patterns in [`diversity_inclusion/urls.py `](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/diversity_inclusion/urls.py) instead of here?

Reply:
Agreed and fixed